host_core/test/support/common.exs

```diff
@@ -137,4 +137,46 @@ defmodule HostCoreTest.Common do
       policy_timeout_ms: 1_000
     }
   end
+
+  # Benchmarking common functions
```

Comment:
Supernit: IMO these should stay in `benchmark/`. Regardless, these helpers are super useful